CruCentralCoast/Database/DatabaseManager.swift

```diff
@@ -33,6 +33,38 @@ class DatabaseManager {
     func getMinistries(_ completion: @escaping ([Ministry])->Void) {
         getData(completion)
     }
+    
+    private func getArticleResources(_ completion: @escaping ([ArticleResource])->Void) {
```

Comment:
Is this method ever used? Same with next two methods

Reply:
I think that was leftover stuff from something else i used to do =P ill get rid of those